docs/index.md

```diff
@@ -20,10 +20,17 @@ To see RedisGraph in action, visit [Demos](https://github.com/RedisGraph/RedisGr
 
 ## Quickstart
 
+1. [Redis Cloud](#rediscloud)
 1. [Docker](#docker)
-2. [Build](#building)
-3. [Start](#loading-redisgraph-into-redis)
-4. [Use from any client](#using-redisgraph)
+1. [Build](#building)
+1. [Start](#loading-redisgraph-into-redis)
+1. [Use from any client](#using-redisgraph)
```

Comment:
How come each line is numbered as `1.` ?

Reply:
it's a more convenient way of creating ordered lists.

Reply:
It renders perfect. https://github.com/RedisGraph/RedisGraph/blob/45590ce5725ad7c5d731d6fdaadbf528994f503d/docs/index.md In the idea of enhancing things every time you touch them :-)